warrior/Framework/ClassUtils/WNetwork/warrior_cli_class.py

```diff
@@ -312,11 +312,15 @@ def print_warn_msg(keyvars, numpats):
                 pNote(save_msg3.format(resp_ref, response))
             elif resp_keys is not None:
                 keys = resp_ref.split(',')
+                # get the patterns from pattern entries in testdata file
                 patterns = [k.get("resp_pattern_req") for k in resp_keys]
+                # warn if number of patterns does not match number of resp_ref keys
                 if len(keys) != len(patterns):
                     print_warn_msg(keys, len(patterns))
                 if inorder:
                     pNote(save_msg1+' inorder.')
+                    # since inorder pattern matching selected, join all the
+                    # patterns in order to create a single big pattern
                     cpatterns = map(lambda s: "(" + s + ")", patterns)
```

Comment:
lambda function can be replaced by list comprehension, which is more readable

Reply:
done.